### PR TITLE
Create PR.md

### PR DESCRIPTION
**Pull request by aiapte**

### CODE REVIEW

**Review by kovacova — COMMENTED** [review]
Well done A, that's a lot of good progress work with nice visualizations! To improve the readability of the document you may consider adding a few extra comments or interpretation of the results. Nice work!

T.

**Review by kovacova — COMMENTED** [review]
## Sprint Challenge Code Review
### Week 17 - Neural Networks
---
#### Part 1 — Definitions
You correctly defined all terms, and your explanations show a strong understanding and mastery of these fundamental Neural Network terms.

---
#### Part 2 — Perceptron Implementation
You implemented a perceptron class, trained it on the training data and updated weights appropriately until the output from the network is very close to the expected training data output (y).

---
#### Part 3 — MLP Implementation
Your MLP implementation correctly initializes weights, takes in inputs, combines inputs and weights as a weighted sum, activates the weighted sum, and passes the activated values onto the next layer. Backpropagation implementation appropriately updates weights in reverse order and leads to performance improvements in the model. Model's loss should decrease and accuracy improve over multiple iterations.

---
#### Part 4 — Keras MLP and Hyperparameter Tuning
Your Keras implementation appropriately uses a sequential model with dense layers, and appropriate architecture with at least one hidden layer, activation functions, appropriate loss function, and accuracy metric. Model's verbose output shows significant improvement from first to last epoch during training. You showed your work in hyperparameter tuning two features and reported the CV accuracy of each parameter combination that is being tested.

<details><summary>Well done, A!</summary>

</details>
